src/net/sourceforge/kolmafia/utilities/StringUtilities.java

```diff
@@ -781,20 +781,31 @@ public static final double parseDouble(String string) {
   }
 
   public static final String basicTextWrap(String text) {
+    return basicTextWrap(text, 80);
+  }
+  /*
+  Overloaded to facilitate hand crafted test strings where it is easier to follow what is happening with
```

Comment:
If adding a comment to the top of a function, better to just go the whole way and make it a jsdoc (with `/**` and the like)